noise_std:
    linear_acceleration: 
        x: 0.2
        y: 0.2
        z: 0.2
    angular_velocity:
        x: 0.1
        y: 0.1
        z: 0.1
    magnetic_field:
        x: 0.1
        y: 0.1
        z: 0.1
    fluid_pressure: 20.0
